dev/cnf/oss_dependencies.maven

```diff
@@ -64,6 +64,9 @@ com.sun.faces:jsf-api:2.2.14
 com.sun.faces:jsf-impl:2.2.14
 com.sun.istack:istack-commons-runtime:4.0.0-M3
 com.sun.istack:istack-commons-tools:4.0.0-M3
+com.sun.mail:jakarta.mail:2.0.0-RC5
+com.sun.mail:javax.mail:1.5.6
+com.sun.mail:javax.mail:1.6.2
```

Comment:
I don't think these two dependencies are needed are they? Only the jakarta.mail one above them

Reply:
those dependencies are needed for javaMail 1.5 and 1.6

Reply:
Ah, it is because Chuck has switched those two to build from source. So they are here instead...
https://github.com/OpenLiberty/open-liberty/blob/7c58e0ffca091023a7e844bc9b32c4a84fd2dcde/dev/cnf/oss_source_dependencies.maven#L6
I'm pretty sure that you don't want to add them back in here but if in doubt, check with @cbridgha

Reply:
Ah ok .